tests/unittest_inference.py

```diff
@@ -4050,6 +4050,42 @@ def inner():
         assert not isinstance(inferred, nodes.ClassDef)  # was inferred as builtins.type
         assert inferred is util.Uninferable
 
+    def test_infer_subclass_attr_instance_attr_indirect(self):
+        node = extract_node(
+            """
+        class Parent:
+            def __init__(self):
+                self.data = 123
+
+        class Test(Parent):
+            pass
+        t = Test()
+        t
+        """
+        )
+        inferred = next(node.infer())
+        # self.assertIsInstance(inferred, bases.Instance)
+        const = next(inferred.igetattr("data"))
+        self.assertIsInstance(const, nodes.Const)
+        self.assertEqual(const.value, 123)
```

Comment:
Can you use `assert isinstance` and `assert .. == ..` instead? We try to move more towards `pytest` and away from `unittest`. After you changed it, you probably need to add `@staticmethod` as well.

Same for the test case below.

Reply:
Sure, I wasn't sure what the prevailing style was at the time so I just tried to match the surrounding tests